### ISSUE TITLE
Add integration with Pytorch Ignite

### ISSUE THREAD

**Issue by mihran113** [aimhubio/aim#1322]
## 🚀 Feature

Add integration with Pytorch Ignite.
### Motivation

Currently Aim does not support out of the box integration with Pytorch Ignite. It would be nice to add adapters for easier metric and hyperparam tracking.

### Pitch

Pytorch Ignite already supports multiple experiment tracking tools (Mlflow, TB and etc.) as loggers, so it should not be hard to create similar logger and handler classes for Aim as well

### Additional context

Below is a link to the source code for `MLflowLogger` from `ignite.contrib.handlers` package for easy reference
https://pytorch.org/ignite/v0.4.8/generated/ignite.contrib.handlers.mlflow_logger.html#module-ignite.contrib.handlers.mlflow_logger
